app/javascript/gobierto_dashboards/webapp/containers/tender/TendersIndex.vue

```diff
@@ -0,0 +1,36 @@
+<template>
```

Comment:
This file is EXACTLY the same as ContractsIndex. That means if should be turn into a shared component, right? Pass them their properties to make them different